code/datums/gamemode/factions/bloodcult/bloodcult_runes.dm

```diff
@@ -439,7 +447,15 @@ var/list/uristrune_cache = list()//icon cache, so the whole blending process is
 		active_spell = null
 
 /obj/effect/rune/proc/fizzle(var/mob/living/user)
-	user.say(pick("B'ADMINES SP'WNIN SH'T","IC'IN O'OC","RO'SHA'M I'SA GRI'FF'N ME'AI","TOX'IN'S O'NM FI'RAH","IA BL'AME TOX'IN'S","FIR'A NON'AN RE'SONA","A'OI I'RS ROUA'GE","LE'OAN JU'STA SP'A'C Z'EE SH'EF","IA PT'WOBEA'RD, IA A'DMI'NEH'LP"))
+	var/silent = 0
+	if (user.mind && user.mind.GetRole(CULTIST))
+		var/datum/role/cultist/C = user.mind.GetRole(CULTIST)
+		for (var/T in C.tattoos)
```

Comment:
See previous remark, you can even do something like `var/silent = ("Silent casting" in C.tattoos)`